app/controllers/orders_controller.rb

```diff
@@ -5,7 +5,11 @@ class OrdersController < ApplicationController
 
   # GET /orders
   def index
-    @orders = Order.all.where(employee: current_employee)
+    @orders = if params.key?(:status)
```

Comment:
I would use something like below:
the thing is - you are checking only if the key is present, but what if params will look like this: 
`params: {status: ''}`

Consider also some validation of incoming status (cen be even checking if it is on the list of allowed values), because it is url param I can change it however I like, what if I pass `Foo` as status?
```suggestion
    @orders = if params[:status].present?
```

Reply:
Thanks for the advice! I added validations and now if you type something other than status, you don't get an error like before :)